src/main/java/uk/gov/hmcts/sptribs/systemupdate/service/CcdSearchService.java

```diff
@@ -37,8 +37,6 @@ public class CcdSearchService {
     @Autowired
     private CoreCaseDataApi coreCaseDataApi;
 
-
-
     public List<CaseDetails> searchForAllCasesWithQuery(
```

Comment:
The parameter latestVersion wasn't used inside the method searchForCasesWithVersionLessThan. Can this be removed?

Reply:
latestVersion seems to be used in BaseMigration & BaseMigrationTest so I changed the SearchSourcebuilder query to use latestVersion Can you check please if that's correct or not.

Reply:
dataVersion field has been retired. And don't know how else we can check the latest version. So, let's leave this like this. This method may need to be removed in future.